alvr/server/src/tracking.rs

```diff
@@ -265,7 +267,89 @@ pub fn to_openvr_hand_skeleton(
             * Quat::from_euler(EulerRot::YXZ, -FRAC_PI_2, FRAC_PI_2, 0.0),
         position: gj[1].position,
     };
-
+    // test pinch
```

Comment:
I believe this is not the right function where to do this. `to_openvr_hand_skeleton` was supposed to be a "pure" function, ie no side effects. You can create another function with the only purpose of detecting and sending pinches.